lib/service_sign_in/README.md

```diff
@@ -0,0 +1,57 @@
+# How to publish Service Sign-in pages
+
+If a Content Designer adds a new page via Github, there's a good chance they'll be an external contributor.
+
+If this is not the case feel free to skip 'How to merge a change from an external contributor' steps.
+
+The examples below are from a [release from the Start Pages team](https://github.com/alphagov/publisher/pull/687)
+
+## How to merge a change from an external contributor
+
+As described in the [Pull Request merging RFC](https://github.com/alphagov/govuk-rfcs/blob/2135a4c9363a499a7182e7594d6c4054422e76a1/rfc-052-pull-request-merging-process.md#a-change-from-an-external-contributor) we have a slightly different process for contributions that are not from people in the alphagov organisation.
+
+The main things to note are that we require extra effort in reviews to make sure we don't accidentally merge something malicious.
+
+1. Get a second reviewer, this can be bi-passed for these pages by going to talk to the Content Designer and confirming their Github account is the one raising the request.
+2. Get Continuous Integration to run their Pull Request so that the build will pass.
+
+### Merging external contribution: Getting the build to pass
+
+You'll need to pull their branch and push it yourself to trigger the build, this relies on the fact that their commit will be the same in the branch you push up. You can then remove this branch after this process is finished.
+
+```bash
+# Make sure you're up-to-date
+git pull
+
+# Fetch their forked branch, replace `thomasjhughes` with their username
+# and `patch-1` if their branch is not called `patch-1`
+git fetch https://github.com/thomasjhughes/publisher patch-1:thomasjhughes-patch-1
+
+git checkout thomasjhughes-patch-1
+
+git push --set-upstream origin thomasjhughes-patch-1
+```
+
+Since you're pushing the same commit up to remote, it will trigger the original PR to update with commit being tested.
+
+Once the build has completed and you've merged the PR, make sure to [clean up your temporary branch](https://github.com/alphagov/publisher/branches)
+
+## Releasing the new page
+
+Once the change is merged, you can release these in two steps.
+
+1. [Deploy publisher](https://docs.publishing.service.gov.uk/manual/deploying.html)
+2. Run the Rake task to publish the new page
+
+### Running the Rake task
+
+Go to the [Jenkins Rake task job (Staging)](https://deploy.staging.publishing.service.gov.uk/job/run-rake-task/build) ([Production](https://deploy.publishing.service.gov.uk/job/run-rake-task/build))
```

Comment:
You can use the params below in the URL for a zero-type experience:

eg `https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION= publisher& MACHINE_CLASS= backend&RAKE_TASK=service_sign_in:publish[check-update-company-car-tax.en.yaml]`

Reply:
Was trying to figure that out, this is awesome thanks Tijmen 👍